netket/models/autoreg.py

```diff
@@ -278,6 +340,32 @@ def _normalize(log_psi: Array, machine_pow: int) -> Array:
     )
 
 
+@dispatch
+def _conditionals_log_psi(model: ARNNDenseModPhase, inputs: Array) -> Array:
+    """
+    Computes the log of the conditional wave-functions for each site if it takes each value
+    for an ARNN model whose modulus and phase are given by different neural networks.
+    """
+
+    x = jnp.expand_dims(inputs, axis=-1)
+
+    x_imag = jnp.copy(x)
```

Comment:
Dian is right. Copies don't really exist in jax. Your code is equivalent, but let's not add to the amount of operations jax has to deal with...
```suggestion
    x_imag = x
```